src/frontend/src/services/personService.js

```diff
@@ -22,13 +22,13 @@ const enhancePersonWithImage = async (person, imgSize = ImageSize.Normal) => {
   return person;
 };
 
-export const fetchPersons = async (filter, page, count) => {
+export const fetchPersons = async (query) => {
```

Comment:
Do like other service functions and use `{}` as the argument

```suggestion
export const fetchPersons = async ({page, count, name}) => {
```